cmd/context.go

```diff
@@ -0,0 +1,239 @@
+		Use:    "create <vcs-type> <org-name> <context-name>",
+		PreRun: initClient,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return createContext(cl, args[0], args[1], args[2])
+		},
+		Args: cobra.ExactArgs(3),
+	}
+
+	force := false
+	deleteContextCommand := &cobra.Command{
+		Short:  "Delete the named context",
+		Use:    "delete <vcs-type> <org-name> <context-name>",
+		PreRun: initClient,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return deleteContext(cl, force, args[0], args[1], args[2])
+		},
+		Args: cobra.ExactArgs(3),
+	}
+
+	deleteContextCommand.Flags().BoolVarP(&force, "force", "f", false, "Delete the context without asking for confirmation.")
+
+	command.AddCommand(listCommand)
+	command.AddCommand(showContextCommand)
+	command.AddCommand(storeCommand)
+	command.AddCommand(removeCommand)
+	command.AddCommand(createContextCommand)
+	command.AddCommand(deleteContextCommand)
+
+	return command
+}
+
+func listContexts(client *client.Client, vcs, org string) error {
+
+	contexts, err := api.ListContexts(client, org, vcs)
+
+	if err != nil {
+		return err
+
+	}
+
+	table := tablewriter.NewWriter(os.Stdout)
+
+	table.SetHeader([]string{"Provider", "Organization", "Name", "Created At"})
+
+	for _, context := range contexts.Organization.Contexts.Edges {
+
+		table.Append([]string{
+			vcs,
+			org,
+			context.Node.Name,
+			context.Node.CreatedAt,
+		})
+	}
+	table.Render()
+
+	return nil
+}
+
+func contextByName(client *client.Client, vcsType, orgName, contextName string) (*api.CircleCIContext, error) {
+
+	contexts, err := api.ListContexts(client, orgName, vcsType)
+
+	if err != nil {
+		return nil, err
+	}
+
+	for _, c := range contexts.Organization.Contexts.Edges {
+		if c.Node.Name == contextName {
+			return &c.Node, nil
+		}
+	}
+
+	return nil, fmt.Errorf("Could not find a context named '%s' in the '%s' organization.", contextName, orgName)
+}
+
+func showContext(client *client.Client, vcsType, orgName, contextName string) error {
+
+	context, err := contextByName(client, vcsType, orgName, contextName)
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Context: %s\n", context.Name)
+
+	table := tablewriter.NewWriter(os.Stdout)
+
+	table.SetHeader([]string{"Environment Variable", "Value"})
+
+	for _, envVar := range context.Resources {
+		table.Append([]string{envVar.Variable, "••••" + envVar.TruncatedValue})
+	}
+	table.Render()
+
+	return nil
+}
+
+func readSecretValue() (string, error) {
+	stat, _ := os.Stdin.Stat()
+	if (stat.Mode() & os.ModeCharDevice) == 0 {
+		bytes, err := ioutil.ReadAll(os.Stdin)
+		return string(bytes), err
+	} else {
+		fmt.Print("Enter secret value and press enter: ")
+		reader := bufio.NewReader(os.Stdin)
+		str, err := reader.ReadString('\n')
+		return strings.TrimRight(str, "\n"), err
+	}
+}
+
+func createContext(client *client.Client, vcsType, orgName, contextName string) error {
+	return api.CreateContext(client, vcsType, orgName, contextName)
```

Comment:
I ran into an error when checking this locally:

```
 $ go run main.go context create github eric-hu foo
Error: Unable to find organization eric-hu of vcs-type foo: Provided argument value `foo' is not member of enum type.
exit status 255
```


It looks like `api.CreateContext` uses a different argument ordering:

https://github.com/CircleCI-Public/circleci-cli/pull/362/files#diff-9f8031b3385a5e9aa0cdd6f639ce2aa0R51